src/palace/manager/search/v5.py

```diff
@@ -266,6 +266,7 @@ def __init__(self) -> None:
         licensepools.add_property("suppressed", BOOLEAN)
         licensepools.add_property("licensed", BOOLEAN)
         licensepools.add_property("medium", keyword())
+        licensepools.add_property("lane_priority_level", INTEGER)
```

Comment:
For my final draft,  what is the recommended way to add a new revision of the search index schema?

Reply:
Should I copy v5.py -> v6.py,  update the revision number and make my changes?

Reply:
Yeah, that was the intention. Adding a new `.py` file for each new version means that you can go back to older schema versions in the case of some sort of disastrous design failure. :slightly_smiling_face: